src/main/universal/res/config/simplify_quant_pass1.jspec

```diff
@@ -432,23 +431,45 @@ class simplify_quant_pass1 {
     (e1 < e2) ==> (\forall* int i; (i \memberof {e3 .. e4}); Perm(e5, (p1 * (e2 - e1))\p2))
   }
 
-  // We keep the expression around, because it must trigger an error when it is not well formed, but we don't use it
-  // We're counting on the backend to not optimize it away if it contains an error
-  axiom read_spreads_l {
-    read * i
+  // The next read-permission rules are a bit tricky because the semantics for wildcard/read in viper is a bit limited.
+  // Some discussion:
+  // - https://github.com/viperproject/silicon/issues/539
+  // - https://github.com/viperproject/silicon/issues/569
+  axiom read_spreads_l_i {
+    Perm(g1, read * i)
```

Comment:
what about negative `i`?

Reply:
That would result in there being no permission for `g1`. I guess we are hiding an error there (since viper wouldn't allow a negative perm anyway, I think?), so that might be a problem.

Reply:
Viper indeed throws an error upon negative permission, so this rule alters the expected semantics. Not sure if this is worth it, but we do need to address the use case of passing read perms to 2 or more threads, I think.

Reply:
Negative permissions are a falsy well-formedness condition (don't know how else to describe it, it's like injectivity in forall*), so I think that `i >= 0 && i > 0 ==> Perm(g1, read)` is semantics-preserving (though it does not preserve the error)

Reply:
Well, I think I managed to rewrite the example to just use permission a la `Perm(contrib[i], 1 \ N)`, so maybe we no longer need that rule after all. Also, it took quite a bit more steps than before, so I'm suspecting that the previous rewrite rules actually caused some unsoundness... If I can also convert the other example I think we should archive these rewrite rules in the wiki, and do some more thinking about how we want to support this use case in vercors.